tests/pyccel/runtest_print.py

```diff
@@ -0,0 +1,45 @@
+# from math import (inf, nan)
```

Comment:
Please save pyccel test files in the `scripts/` folder

Reply:
done.